src/config/Config.zig

```diff
@@ -1639,7 +1639,10 @@ keybind: Keybinds = .{},
 /// The default value is `detect`.
 @"shell-integration": ShellIntegration = .detect,
 
-/// Shell integration features to enable if shell integration itself is enabled.
+/// Shell integration features to enable. These are applied even when automatic
+/// shell integration is inactive unless explicitly disabled with
+/// `shell-integration-features = false`.
```

Comment:
One more suggestion after re-reading this:

> Shell integration features to enable. These require our shell integration
> to be loaded, either automatically via `shell-integration` or manually.

I think that's clear, and we talk about turning all features off in the next paragraph.